tests/stubs/elm

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env zsh
 
```

Comment:
I think better to set a default version if no value was provided in `SPACESHIP_MOCK_ELM_VERSION `.